src/vscodeTasksAdapter.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should this be in the 'src' folder or the 'test' folder?

Reply:
fixed. I created this directory structure:
- test
  - unitTests
  - integrationTests
    - testAssets